utils/docker/entrypoint.sh

```
@@ -26,7 +33,12 @@ else
   exec \
     mehari \
     server run \
-      --path-db "$PATH_DB" \
+      $(test -e "$PATH_TRANSCRIPTS_37" && echo --transcripts "$PATH_TRANSCRIPTS_37") \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)